### DISCUSSION TITLE
Announcing: Slice by :pizza: , Updates to issue forms, Group menu, plus bug fixes and improvements :dizzy:

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#63381] [Projects and Issues]
Today's Issues & Projects update brings you the brand new slice by, updates to issue forms, and a group menu across layouts!

## 🍕 Slice by 

You can now slice by field values in your project views! Select a `Slice by` field from the view configuration menu to pull the field values into a panel on the left, and click through the values in the list to adjust the items shown in the view on the right.

![slice_by](https://user-images.githubusercontent.com/2180038/259850616-e8bed780-945e-475d-bb9c-4659b6bd2d34.jpg)

Configuring a `Slice by` field allows you to quickly toggle through and review your project items, saving you additional filters and views to quickly understand the state and details of your project. While you can slice by your issue and project fields, some helpful slice by views include:
- Single select fields to view items broken up by status, priority, or focus area
- `Labels` to group items by repository labels
- `Assignees` to see who is working on what

## 📋 Updates to issue forms

You can now configure custom issue forms to automatically add an issue to a project as well as set defaults for drop downs by adding a YAML form definition file to the `/.github/ISSUE_TEMPLATE` folder in your repository.

![issue form yaml syntax](https://github.com/github/release-assets/assets/98360703/6917b6b1-314e-438d-afe2-6566768133c1)

##  Group menu for item and group actions

We've added a group menu to quickly take action on items in a group or on the group itself. Click `...` from the group header on your tables, boards, or roadmaps to archive or delete all items in a group, edit the group details directly, hide the group from the view, or delete it from your project.

![group menu](https://user-images.githubusercontent.com/2180038/259850868-a5177f08-bbe7-4100-a49e-ddea63157dd4.jpg)

## :sparkles: Bug fixes and improvements
* Fixed keyboard navigation and focus when navigating to `Make a copy` from the project `...` menu
* Fixed group header spacing on the roadmap layout
* Fixed a bug where using the `@next` filter qualifier for iteration fields was referencing the incorrect iteration
* Fixed a bug with the numerical `Field sum` decimal precision

What do you think of these updates? Have questions, comments, or excitement, share it below ⬇️ .

**Comment by justinretzolk** [comment]
I'm really liking `Slice by`, and it's gotten me thinking about some neat new ways to organize. As a minor UI enhancement, I'd love the option to hide options in the `Slice by` side panel without filtering them out entirely, particularly when slicing by `Labels`. My use case is:

We have a number of different services in our project, issues pertaining to which get a `service/<service_name>` label (along with many potential other labels). My immediate thought was to create a service-level bug tracking board view by filtering by `label:bug`, and slicing by `Labels`. This works, but all of the other labels that might be on a given issue also appear.

**Reply by rileybroughten** [reply]
If you add a filter for the label on the view, it will only show the items with that filter. However, any other labels that are attached to those items will still appear in the slice panel. Is that not the case for you? Want to make sure I'm understanding your use case correctly. Thank you!

<img width="1440" alt="Screenshot 2023-08-17 at 12 17 27 PM" src="https://github.com/community/community/assets/101840513/256854bb-4ce3-441f-a9ab-34745c5442cd">

**Reply by justinretzolk** [reply]
Hey @rileybroughten 👋 Thanks for getting back so quickly! That functionality is working as expected. My use-case in particular, I don't want to remove items that have those labels, I just don't need to see those labels in the left hand column.

Here's a quick screen grab from the Project that I'm using to test this out. Essentially, I want to see all issues that have `bug` and at least one `service/*` label, and then to be able to use the left side bar to look at the bugs related to a particular service (by clicking on the relevant `service/<label_name>`). If I were to add `-label:documentation`, the bug that appears on line 15 in the screenshot would disappear from this view.

Perhaps I'm misusing the feature to some degree, but it would definitely help my specific use-case 😅 

<img width="1876" alt="Screen Shot 2023-08-17 at 15 19 49" src="https://github.com/community/community/assets/44710313/565a3c15-cdd5-441a-abd3-cd08989dc354">

**Reply by rileybroughten** [reply]
I understand what you're trying to do here, unfortunately with how the filtering works with the panel, there's not a way to hide specific labels other than filtering them out completely. I'll note it for feedback though, I can see how it would be helpful especially with long lists!

**Comment by tellisnz** [comment]
> You can now configure custom issue forms to automatically add an issue to a project as well as set defaults for drop downs by adding a YAML form definition file to the /.github/ISSUE_TEMPLATE folder in your repository.

This is great - but could it also be possible to link issue form fields to custom fields in the project so they are set on the form too?

**Reply by rileybroughten** [reply]
You can configure the `Item added to project` workflow within each project to make sure the status is set correctly when new items are added - check out the [documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/using-the-built-in-automations) for more details!

**Reply by tellisnz** [reply]
Yeah thanks @rileybroughten we do have that one for status.

We also have a number of other custom fields we would like to set directly from the issue form - it'd be great if this was available to map directly from issue form field -> issue project custom field, without having to [create actions](https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/automating-projects-using-actions) to do this ourselves
